data/templates/responsive/elements/common.xsl

```diff
@@ -164,10 +164,39 @@
         </tr>
     </xsl:template>
 
-    <xsl:template match="tag[@name = 'license' or @name = 'link' or @name = 'see' or @name = 'uses' or @name = 'author']" mode="tabular">
+    <xsl:template match="tag[@name = 'see' or @name = 'uses']" mode="tabular">
+        <xsl:variable name="url">
+            <xsl:choose>
+                <xsl:when test="not(contains(@link, 'http'))">
```

Comment:
I expect this to fail when an referenced element contains 'http', I think you need to check if the link is a valid path for the extension used below. You might need to add an extra method to that class.